### PR TITLE
Luis Authoring v3.0-preview

### PR DESCRIPTION
**Pull request by nebadr**
LUIS Authoring v3.0-preview APIs

### CODE REVIEW

File: sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj [hunk 1]
```diff
@@ -2,14 +2,14 @@
   <PropertyGroup>
     <AssemblyTitle>Microsoft Azure Cognitive Services Language LUIS Authoring</AssemblyTitle>
     <Description>Provides API functions for consuming the Microsoft Azure Cognitive Services LUIS Authoring API.</Description>
-    <Version>3.2.0-preview.1</Version>
+    <Version>3.8.0-preview</Version>
```
Comment (AlexGhiondea) [inline sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj]: This is quite a jump -- should this be `3.3.0-preview.1` ?

File: sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/Features.cs [hunk 2]
```diff
@@ -98,17 +94,9 @@ public async Task<HttpOperationResponse<IList<PatternFeatureInfo>>> ListApplicat
             {
                 throw new ValidationException(ValidationRules.CannotBeNull, "versionId");
             }
-            if (skip < 0)
```
Comment (AlexGhiondea) [inline sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/Features.cs]: Why remove the validation?

**Review by AlexGhiondea — APPROVED** [review]

File: sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj (hunk 1 above)
Comment (nebadr) [reply sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj]: @AlexGhiondea We got this feedback on our Runtime nuget, that when we release a preview nuget it should be the next .8 version.

Comment (nebadr) [reply sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj]: Here is the comment: https://github.com/Azure/azure-sdk-for-net/pull/5991#issuecomment-487732904

File: sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/Features.cs (hunk 2 above)
Comment (nebadr) [reply sdk/cognitiveservices/Language.LUIS.Authoring/src/Generated/Features.cs]: It's not removed, that's another api. The api with skip and take still has their validation

File: sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj (hunk 1 above)
Comment (weshaggard) [reply sdk/cognitiveservices/Language.LUIS.Authoring/src/Microsoft.Azure.CognitiveServices.Language.LUIS.Authoring.csproj]: That looks to be based on older versioning guidance. We have been updating our guidance to follow https://github.com/Azure/azure-sdk-for-net/blob/master/doc/dev/Versioning.md. I'd suggest you updating the version to be `3.2.0.preview.2` unless there are enough changes to jump to a new minor version.

**Pull request merged**